mconfig

```diff
@@ -697,15 +698,25 @@ cat $makeit_makefile_base >> $makeit_makefile
 drawline $makeit_fragsdir/common_opts.mk
 cat $makeit_fragsdir/common_opts.mk >> $makeit_makefile
 
+drawline $makeit_fragsdir/go_common_opts.mk
+cat $makeit_fragsdir/go_common_opts.mk >> $makeit_makefile
+
+if [ -f "$makeit_fragsdir/go_${host}_opts.mk" ]; then
+	drawline $makeit_fragsdir/go_${host}_opts.mk
+	cat $makeit_fragsdir/go_${host}_opts.mk >> $makeit_makefile
+fi
 
 if [ "$appsec" = "1" ]; then
 	drawline $makeit_fragsdir/go_appsec_opts.mk
 	cat $makeit_fragsdir/go_appsec_opts.mk >> $makeit_makefile
-else
-	drawline $makeit_fragsdir/go_common_opts.mk
```

Comment:
Instead of making this either/or, moved the common up above and now we just set `appsec` options if appsec is set.